projects/cdk/classes/assert.ts

```diff
@@ -8,3 +8,12 @@ export const tuiAssert = {
             : EMPTY_FUNCTION;
     },
 };
+
+export function tuiAssertIsHTMLElement(element: unknown): asserts element is HTMLElement {
+    const isElement =
+        element instanceof HTMLElement ||
```

Comment:
It should only check for `HTMLElement`. Also let's change it to something along the lines of:
```ts
export function tuiAssertIsHTMLElement(node: Node | EventTarget): asserts node is HTMLElement {
  const isHTMLElement = node instanceof node.ownderDocument?.defaultView?.HTMLElement;

  tuiAssert.assert(isHTMLElement, 'Node is not an HTMLElement');
}
```
This in theory should help against `iframe` classes but I didn't check.

Reply:
<img width="673" alt="image" src="https://user-images.githubusercontent.com/12021443/163876007-17a58b19-4a90-4aab-ae68-370e02d7abf7.png">

Reply:
fixed